packages/suite/src/views/wallet/trading/common/TradingDetail/TradingDetailSurvey.tsx

```diff
@@ -0,0 +1,93 @@
+import { useMemo } from 'react';
+
+import {
+    Feature,
+    TradingSurveyPayload,
+    resolveMessageContent,
+    selectFeatureConfig,
+    validateTradingSurvey,
+} from '@suite-common/message-system';
+import { Button, Card, Column, Paragraph, Text } from '@trezor/components';
+import { spacings } from '@trezor/theme';
+
+import { goto } from 'src/actions/suite/routerActions';
+import { useDispatch, useSelector, useTranslation } from 'src/hooks/suite';
+import { selectLanguage, selectTorState } from 'src/selectors/suite/suiteSelectors';
+import { getTorUrlIfAvailable } from 'src/utils/suite/tor';
+
+export const safeValidateTradingSurvey = (data: unknown): TradingSurveyPayload | null => {
```

Comment:
It seems to be used only locally:
```suggestion
const safeValidateTradingSurvey = (data: unknown): TradingSurveyPayload | null => {
```